pull_request_template.md

```diff
@@ -1,18 +1,9 @@
-## What did you do? 
-- [describe the proposed changes]
-- 
-
-
-## Why did you make this change?
-(Cite Issue number OR provide rationalization of changes if no issue exists)
-(If fixing a bug, please add any relevant error or traceback)
-
+(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the change. The "why" may simply be citing a relevant Galaxy issue.) 
```

Comment:
```suggestion
(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the changes. The "why" may simply be citing a relevant Galaxy issue.)
(If fixing a bug, please add any relevant error or traceback)
(For UI components, it is recommended to include screenshots or screencasts)
```

Reply:
I removed the fixing a bug language because it felt redundant (on an admittedly spiritual level) with the request to cite an issue - but I'll restore it.

Reply:
Not every bug has an issue, and I don't think we should open one just for the sake of mentioning it in a PR description. Thanks for applying my suggestions!